dm/dm/worker/server_test.go

```diff
@@ -640,3 +664,80 @@ func loadSourceConfigWithoutPassword(c *C) *config.SourceConfig {
 	sourceCfg.From.Password = "" // no password set
 	return sourceCfg
 }
+
+func (t *testServer) testSourceWorker(c *C) {
```

Comment:
why move it? the content of this function is related to source worker, so it's natural to put it in source_worker_test.go